#!/usr/bin/env python

PACKAGE = "robdos_sim"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("ori_kp", double_t, 0, "ori_KP", 900, 0, 2000)
gen.add("ori_kd", double_t, 0, "ori_KD", 250, 0, 1000)
gen.add("ori_ki", double_t, 0, "ori_KI", 0.0, 0, 1000)
gen.add("ori_wu", double_t, 0, "ori_WU", 0.0, 0, 1000)

gen.add("thr_orientation", double_t, 0, "thr_orientation", 10.0, 0, 90)

exit(gen.generate(PACKAGE, "robdos_sim", "controllerOrientation"))